docs/extending/developing-code/extending/case-studies/KroneckerProduct/checking.rst

```diff
@@ -0,0 +1,58 @@
+Checking the ``KroneckerProduct[]`` Skeletal Version
+====================================================
+
+As before with ``Undefined`` although there is code to wrote, we have
+entered enough to be able to see that the builtin has been added, and
+that we can get some information about it.
+
+So again, let us try this much. Fire up a a version Django interface that has
+access to the *development* version of mathics-core
+
+.. code-block::
+
+    $ cd Mathics3/mathics-django
+    $ make runserver
+    Watching for file changes with StatReloader
+    Performing system checks...
+
+    System check identified no issues (0 silenced).
+    August 21, 2022 - 01:19:05
+    Django version 4.0.5, using settings 'mathics_django.settings'
+    Starting development server at http://127.0.0.1:8000/
+
+Connect to ``http://127.0.0.1:8000`` or the address above and go into
+the "about" section and see that you are using right version of
+mathics core.
+
+
+Finally click the "?" at the extreme right and type "Undef". You should
```

Comment:
Why "Undef" ?

Reply:
That is the smallest prefix of "Undefined" that will you to the "Undefined" page. I'll expand on this when I have time to work on this again.